rpcs3/Emu/Io/LogitechG27.cpp

```diff
@@ -746,8 +748,8 @@ void usb_device_logitech_g27::interrupt_transfer(u32 buf_size, u8* buf, u32 endp
 
 		// calibrated, unsure
 		set_bit(buf, 82, true);
-		// shifter connected
-		set_bit(buf, 83, true);
+		// shifter connected, 0 for now until it can be verified
+		set_bit(buf, 83, false);
 		// shifter stick down
 		set_bit(buf, 86, shifter_1 || shifter_2 || shifter_3 || shifter_4 || shifter_5 || shifter_6 || shifter_r);
```

Comment:
```suggestion
		set_bit(buf, 86, shifter_r);
```

Reply:
>So we likely should add an individual mapping for it, since you've confirmed that having a gear locked in doesn't hold bit 86 down.

There is no use to press the shifter down unless you're engaging the reverse gear. They can go together.

>Would you like to make your own pull request to add that?

You can do it.

>Also byte 8 and byte 9 does look like analog coordinate of the gear stick

Yes, the lsb are just analog noise and oscillates between iterations